### PR TITLE
attackables.h has an Attackable class, for now the class of which pla…

### PR DESCRIPTION
**Pull request by TinyAtoms**
…yers and monsters can be subclassed from

attackables.cpp has the member functions of Attackable
helper has the Helper class.
helper.cpp has the Helper member functions.

### CODE REVIEW

File: helper.cpp [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    int total = 0;
+    for (int i = 0; i < amount; i++) {
+        int roll = std::rand() % sided + 1;
+        total += roll;
+    }
+    return total;
+}
+
+// totally proof of concept, this is not how damage will be generated
+Attackable &Helper::battle(Attackable &attacker, Attackable &attackee) {
```
Comment (Hello-User) [inline helper.cpp]:
```suggestion
Attackable &Helper::battle(Attackable &attacker, Attackable &target) {
```

File: attackables.h [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+
+class Attackable {
+public:
+    std::string a_name;
+    int health;
+    int strength;
+    int armour;
+
+    Attackable(std::string name, int hp, int str, int arm);
```
Comment (Hello-User) [inline attackables.h]:
```suggestion
    Attackable(std::string,int,int,int);
    
    Double check this, but I don't think we need to specify a name here, only when we call the function ( in attackables.cpp)
```

Comment (TinyAtoms) [reply attackables.h]: Yeah, I thought I removed that. Will do so

**Comment by TinyAtoms** [comment]
Damn, I accidentally added IDE files too...

**Pull request merged**